src/components/RangeSliderContainer.ts

```diff
@@ -160,24 +167,38 @@ export default class RangeSliderContainer extends Component<RangeSliderContainer
         };
     }
 
-    private handleAction(value: number) {
+    private handleChangeAction(value: number) {
         if ((value || value === 0) && this.props.mxObject) {
-            this.executeMicroflow(this.props.onChangeMicroflow, this.props.mxObject.getGuid());
+            this.executeOnChangeAction(this.props.mxObject);
         }
     }
 
-    private executeMicroflow(actionname: string, guid: string) {
-        if (actionname) {
-            window.mx.ui.action(actionname, {
+    private executeOnChangeAction(mxObject: mendix.lib.MxObject) {
+        const { onChangeMicroflow, onChangeNanoflow, mxform } = this.props;
+        if (onChangeMicroflow) {
+            window.mx.ui.action(onChangeMicroflow, {
                 error: (error) => window.mx.ui.error(
-                    `An error occurred while executing microflow: ${actionname}: ${error.message}`
+                    `An error occurred while executing microflow: ${onChangeMicroflow}: ${error.message}`
                 ),
                 params: {
                     applyto: "selection",
-                    guids: [ guid ]
+                    guids: [ mxObject.getGuid() ]
                 }
             });
         }
+
+        if (onChangeNanoflow.nanoflow) {
+            const context = new mendix.lib.MxContext();
+            context.setContext(mxObject.getEntity(), mxObject.getGuid());
+            window.mx.data.callNanoflow({
+                context,
+                error: error => window.mx.ui.error(
+                    `An error occurred while executing nanoflow: + ${onChangeNanoflow} : ${error.message}`
```

Comment:
The `onChangeNanoflow` is an object and should be placed inside the string of an error message.
Unfortunate the NF name is not available in the runtime. So for now we should 
refer the XML configuration property name. In this case `On change nanoflow`

``` ts
`An error occurred while executing the on change nanoflow: ${error.message}`
```

Reply:
Done